### PR TITLE
Update NETStandard docs for 1.6

### PR DESCRIPTION
**Pull request by ericstj**
Update the framework mapping to include NS 1.6.

Update contract support table with latest generated result.

/cc @wesh @danmosemsft

### CODE REVIEW

File: Documentation/architecture/net-platform-standard.md [hunk 1]
```diff
@@ -48,23 +48,24 @@ Each platform has it own rules for how it anchors certain assemblies. In .NET Fr
 In general, class libraries which target a lower .NET Platform Standard version, like 1.0, can be loaded by the largest number of existing platforms, but will have access to a smaller set of APIs. On the other hand, class libraries which target a higher .NET Platform Standard version, like 1.3, can be loaded by a smaller number of newer platforms, but will have access to a larger, more recent set of APIs.
 
 | Target Platform Name | Alias |  |  |  |  |  | |
```
Comment (vcsjones) [inline Documentation/architecture/net-platform-standard.md]: There needs to be another `  |` here or the 1.6 column won't display.

**Comment by stephentoub** [comment]
@dotnet-bot test this please

File: Documentation/architecture/net-platform-standard.md (hunk 1 above)
Comment (ericstj) [inline Documentation/architecture/net-platform-standard.md]: Good catch.  My markdown editor missed this.

**Comment by weshaggard** [comment]
LGTM

**Comment by stephentoub** [comment]
LGTM2

**Pull request merged**